File: src/main/java/com/snack/snackproject/Beans/Produit.java
package com.snack.snackproject.Beans;

public class Produit {

    private Long id;
    private String nom;
    private String description;
    private float prix;


    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) { this.nom = nom; }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) { this.description = description; }

    public float getPrix() {
        return prix;
    }

    public void setPrix(float prix) { this.prix = prix; }
}

File: snack-app/src/app/produit/produit.ts
export class Produit {

  id: number;
  nom: string;
  description: string;
  prix: number;

  constructor(id: number, nom: string, description: string, prix: number){
    this.id = id;
    this.nom = nom;
    this.description = description;
    this.prix = prix;
  }

}
